Makefile

```diff
@@ -153,42 +153,38 @@ endif
 # --- Main target variable definitions ----------------------------------------
 #
 
-MK_BASE_FLAMEC_SRC                    :=
-MK_BASE_FLAMEC_OBJS                   :=
+MK_BASE_FLAMEC_SRC                          :=
+MK_BASE_FLAMEC_OBJS                         :=
 
-MK_BLAS_FLAMEC_SRC                    :=
```

Comment:
Can you fix those whitespace  changes?

Reply:
Fixed:  https://github.com/iotamudelta/libflame/pull/4/commits/959b92826d2a5e688f739ac96c83d73139158a39

Reply:
looks like there is still a single whitespace off.....

Reply:
fixed? https://github.com/iotamudelta/libflame/pull/4/commits/6c46c7ba244bcde2c0c4896f2cd9ed9b70aee18b